File: server/src/main/kotlin/org/bitlap/server/raft/cli/BSQLException.kt
package org.bitlap.server.raft.cli

import org.bitlap.common.exception.BitlapException

/**
 * driver exception
 *
 * @author 梦境迷离
 * @since 2021/6/6
 * @version 1.0
 */
data class BSQLException(val msg: String = "SQL Exception") : BitlapException(msg)

File: bitlap-client/src/main/scala/org/bitlap/jdbc/BSQLException.scala
/* Copyright (c) 2022 bitlap.org */
package org.bitlap.jdbc

import java.sql.SQLException

/** @author
 *    梦境迷离
 *  @since 2021/11/20
 *  @version 1.0
 */
case class BSQLException(
  msg: String,
  state: String = null,
  code: Int = -1,
  cause: Throwable = null
) extends SQLException(msg, state, code, cause)
